coforma/swift-tech-challenge
# [#14] Add tooling for unit, integration, and accessibility tests

## Summary

- Configures tooling for all testing types (unit, integration, accessibility)
- Adds utilities for all testing types
- Adds baseline tests for all testing types

Addresses #14

## Type of change

## How To Test
1. Pull changes
2. Run `yarn`
3. Run unit tests (`yarn test:cov`) and verify that all unit tests pass
4. Spin up locally
5. Run integration tests (`yarn test:cy:open`) and verify that all integration tests pass

## Checklist:
- ~If there are security concerns they are addressed or ticketed after being discussed~
